pkg/controller/loadtest.go

```diff
@@ -275,7 +274,7 @@ func (c *Controller) processNextWorkItem() bool {
 // converge the two. It then updates the Status block of the LoadTest resource
 // with the current status of the resource.
 func (c *Controller) syncHandler(key string) error {
-	ctx, cancel := context.WithTimeout(context.Background(), kubekangal.KubeTimeout)
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
Why do we have fixed time here now? Seems like a magic const now.

Reply:
It should never have used "kube timeout" in the first place, so we can either remove it or set a new duration for it.  
It can be labelled as `syncTimeout` constant if that helps.

Reply:
This topic is continued in https://github.com/hellofresh/kangal/pull/106#discussion_r563588998